File: Delete occurrences of an element if it occurs more than n times.js
function deleteNth(arr,n){
  // ...
  a = []  //resultant array
  
  //Counts the occurrence of a number in the resultant array
  function counter(num) {
    var count = 0;
    for(let i = 0; i < a.length; i++) {
      if(a[i] === num) {
        count++
      }
    }
    return count
  }
  
  //Push number to a if it occurs less than n times
  for(let j = 0; j < arr.length; j++) {
    if(counter(arr[j]) < n) {
      a.push(arr[j])
    }
  }
  
  return a
  
}

File: Delete occurrences of an element if it occurs more than n times.py
def delete_nth(order,max_e):
    # code here
    a = {}
    b = []
    
    for i in order:
        if(i in a):
            if(a[i] < max_e):
                a[i] += 1
        else:
            a[i] = 1
    
    #add the number to the list if there are less than max_e occurrences within the same list
    for j in order:
        if((a[j] <= max_e) and (b.count(j) < max_e)):
            b.append(j)
    
    return b
